metau-capstone/ScheduleViewController.m

```diff
@@ -39,4 +42,74 @@ - (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
 }
 */
 
+- (nonnull __kindof UICollectionViewCell *)collectionView:(nonnull UICollectionView *)collectionView cellForItemAtIndexPath:(nonnull NSIndexPath *)indexPath {
+    NSNumber *cellNum = @(indexPath.row+1);
+    ScheduleCell *cell = [self.scheduleCollection dequeueReusableCellWithReuseIdentifier:@"ScheduleCollectionCell" forIndexPath:indexPath];
+    cell.dayNum.text = [NSString stringWithFormat:@"%ld", indexPath.row+1];
+    cell.layer.borderColor = [UIColor blackColor].CGColor;
+    cell.layer.borderWidth = 1;
+    
+    // Query for user's day
+    PFQuery *queryForDay = [PFUser query];
+    PFUser *const user = [PFUser currentUser];
+    [queryForDay getObjectInBackgroundWithId:user.objectId block:^(PFObject * _Nullable object, NSError * _Nullable error) {
+        if (!error) {
+            // Highlight current day
+            if ([object[@"userDay"] isEqualToNumber:cellNum]) {
+                cell.dayNum.backgroundColor = [UIColor yellowColor];
+            } else {
+                cell.dayNum.backgroundColor = [UIColor clearColor];
+            }
+        }
+    }];
+    
+    //Query for the day's levels
+    PFQuery *queryForLevels = [PFQuery queryWithClassName:@"Schedule"];
```

Comment:
This is a good example where having this information live on device might make things a bit smoother.